scripts/zones/The_Boyahda_Tree/mobs/Aquarius.lua

```diff
@@ -11,6 +11,14 @@ entity.onMobInitialize = function(mob)
     mob:setMobMod(xi.mobMod.ADD_EFFECT, 1)
 end
 
+entity.onMobSpawn = function(mob)
+    mob:setMod(xi.mod.SLEEPRES, 100)
+    mob:setMod(xi.mod.DELAY, 1000)
+    mob:addMod(xi.mod.DEFP, 50)
+    mob:addMod(xi.mod.ACC, -50)
```

Comment:
Evidence for such low accuracy?

Reply:
Honestly... I wonder if he should be even less accurate. Do we have the ability to get more accurate numbers?

Reply:
haha honestly no clue. It just feels strange for putting some arbitrary number. But I supposed it's the best we can do for now, eh?